docs/job_defines_format.md

```diff
@@ -0,0 +1,43 @@
+# Jenkins Job Definitions
+
+Due to a growing need of having extra metadata available to jenkins consumers
+(both users and automation) a job define system was added to SCT jenkins generator.
+
+It replaces previous `_display_name` file (which used to contain just the display
+ name of the folder) with a special .yaml file called `_jobDefines.yaml`.
```

Comment:
let's stick to snack case for filenames, and replace the `defines` with `definitions`
also it's folder definitions
 
`_folder_definitions.yaml`

Reply:
Okay